lib/vdsm/virt/livemerge.py

```diff
@@ -364,12 +364,17 @@ def _start_commit(self, drive, job):
         try:
             base_target = drive.volume_target(job.base, actual_chain)
             top_target = drive.volume_target(job.top, actual_chain)
+            base_path = drive.volume_path(job.base)
+            top_path = drive.volume_path(job.top)
         except VolumeNotFound as e:
             self._untrack_job(job.id)
             raise exception.MergeFailed(
                 str(e), top=job.top, base=job.base, chain=actual_chain,
                 job=job.id)
 
+        # Prune stale bitmaps from base volume.
+        self._dom._vm.prune_bitmaps(base_path, top_path)
```

Comment:
Few issues:
- This cannot be sued for active layer merge, since bitmaps created while the vms is running were not written to disk yet, and bitmaps that were on disk are all with the in-use flag.
- Like cold merge, it is more future proof to prune bitmaps before we measure the volume, not right before commit.
- We should pass domain, image and volume uuids to storage apis, not paths.

Reply:
I'll split this commit into another PR as I rework it.